requirements/requirements.txt

```diff
@@ -10,5 +10,5 @@ contexttimer
 ninja
 torch>=1.11
 safetensors
-flash_attn>=2.0
+flash-attn==2.0.5
```

Comment:
Remove this from requirements, as it's not essential.

Reply:
done